### PR TITLE
Springdoc logger

### LINKED ISSUE #6996: Springdoc logger
When adding the springdoc module, we should change this log level:

```xml
<logger name="io.swagger.v3.core.converter.ModelConverterContextImpl" level="WARN" />
```

### PR DESCRIPTION
**Pull request by qmonmert**
Fix #6996

### CODE REVIEW

File: src/main/java/tech/jhipster/lite/generator/server/springboot/apidocumentation/springdoccore/domain/SpringdocModuleFactory.java [hunk 1]
```diff
@@ -58,6 +59,8 @@ private JHipsterModule buildModule(JHipsterModuleProperties properties, SpringDo
         .set(propertyKey("springdoc.swagger-ui.tryItOutEnabled"), TRUE)
         .set(propertyKey("springdoc.enable-native-support"), TRUE)
         .and()
+      .springMainLogger("io.swagger.v3.core.converter.ModelConverterContextImpl", LogLevel.WARN)
```
Comment (DamnClin) [inline src/main/java/tech/jhipster/lite/generator/server/springboot/apidocumentation/springdoccore/domain/SpringdocModuleFactory.java:62]: Missing tests in module factory test for that :)

Comment (qmonmert) [reply src/main/java/tech/jhipster/lite/generator/server/springboot/apidocumentation/springdoccore/domain/SpringdocModuleFactory.java:62]: indeed, I add the test but something is missing?

Comment (DamnClin) [reply src/main/java/tech/jhipster/lite/generator/server/springboot/apidocumentation/springdoccore/domain/SpringdocModuleFactory.java:62]: LGTM

Comment (DamnClin) [reply src/main/java/tech/jhipster/lite/generator/server/springboot/apidocumentation/springdoccore/domain/SpringdocModuleFactory.java:62]: Just need a passing test :D

Comment (qmonmert) [reply src/main/java/tech/jhipster/lite/generator/server/springboot/apidocumentation/springdoccore/domain/SpringdocModuleFactory.java:62]: I found `, logbackFile(), testLogbackFile()` was missing :)

**Pull request merged**